MekHQ/data/rat/(3026) House Arano/Aurigan Coalition and Aurigan Directorate/Aerospace/Heavy/3026 Aurigan Coalition and Aurigan Directorate Aerospace - Heavy.txt

```diff
@@ -0,0 +1,12 @@
+3026 Aurigan Coalition and Aurigan Directorate Aerospace - Heavy
+Stuka STU-K5,1
+Eagle EGL-R6,2
```

Comment:
Why bother having multiple entries for the same craft? Why not just add the numbers upp?

Reply:
I followed how pre-existing RATs were formatted, and the ones I saw had this order

Reply:
Where did this data come from then? Sorry, not really familiar with the Arano sourcebook.

Reply:
House Arano uses standard RAT formatting, as far as I can tell (I've only seen a few RATs, XOTL, one of the Historicals in roughly the same time period, and some FS ones)

Reply:
2d6 rolls, and the aerospace (for this example) are in that order

Reply:
Ok, if the sourcebook has a bunch of double entries for the same mechs, we'll just leave it as is then.